### DISCUSSION TITLE
Disable logs

### DISCUSSION THREAD

**Discussion by RobertoBottino** [javers/javers#1193] [Q&A]
Hello to all, i need help regarding javers library logs.
I was unable to disable them. I use javers for object comparing and all the times it flood with logs.
I have tried to disable them from log4j xml, from logback, from spring boot application.properties.
None of the approaches works.

If someone can tell me the right way to disable them, i'm pretty happy.

Thank you, have a nice day to all.

**Comment by bartoszwalacik** [comment]
Hi, Javers uses standard logback config, see https://www.baeldung.com/logback
https://github.com/javers/javers/blob/master/javers-core/src/test/resources/logback-test.xml
